docs/source/testing/larva/options.rst

```diff
@@ -0,0 +1,265 @@
+     - Replaces all matching occurrences of provided key with IGNORE  
+   * - ignoreContentBetweenKeys 
+     - Replaces all characters found between key1 and key2 with IGNORE
+   * - ignoreKeysAndContentBetweenKeys
+     - Replaces all characters found between key1 and key2, including key1 and key2, with IGNORE
+   * - ignoreContentBeforeKey
+     - Ignores everything before provided key has been found
+   * - ignoreContentAfterKey
+     - Ignores everything after provided key has been found
+   * - ignoreCurrentTimeBetweenKeys       
+     - Ignore current time between key1 and key2
+   * - replaceRegularExpressionKeys
+     - Replaces all matches of regular expression (key1) with the value of key2
+   * - replaceKey
+     - Replaces all matches occurrences of provided key with provided value
+   * - replaceEverywhereKey     
+     - Replaces all matches occurrences of provided key with provided value, same as replaceKey
+   * - removeRegularExpressionKey
+     - Replaces all matches of regular expression with “”
+   * - removeKeysAndContentBetweenKeys
+     - Removes all characters found between key1 and key2, including key1 and key2
+   * - removeKey 
+     - Removes all matching occurrences of provided key
+   * - decodeUnzipContentBetweenKeys
+     - Decode and unzip content between key1 and key2
+   * - canonicaliseFilePathContentBetweenKeys
+     - Canonicalise filepath content between key1 and key2
+   * - formatDecimalContentBetweenKeys
+     - Format decimal content between key1 and key2
+
+
+Usually those options are written in the ``common.properties`` file of your Larva test, in this way they apply to all test scenarios which have included this ``common.properties`` file. If you want to use options in a particular test scenario, then you can put them in that related ``scenario.properties`` file.
+
+Some options require 2 keys to indicate start and end, which are denoted as key1 and key2. Both require the same identifier. The identifier can be either an increasing sequence of integers starting at 1 for each option or an string starting with a “.”. The text indicates what to look for.
+
+Imagine that you want to compare the following result with the stored result:
+
+.. code-block:: XML
+
+    <Result>
+      <RecordID> 12</RecordID>
+      <FileName>C:\Users\..\zipfile</FileName>
+      <File>zipfile.zip</File>
+      <Decimal>12.34</Decimal>
+      <Category>ABC</Category>
+      <Timestamp>2019-10-02T10:12:43.788+0100</Timestamp>
+      <Message>In this record, RecordID='12', RecordID and Timestamp change every time when you run the test</Message>
+    </Result>
+
+Using ignore
+----------------------
+
+**ignoreRegularExpressionKey**
+
+This replaces all occurrences of regular expression with IGNORE. By writing  
+
+.. code-block::
+
+  ignoreRegularExpressionKey1.key="^\d+$"
+
+all the content of this current result will be ignored, so no matter what’s in it, the current result will always pass the test.
+
+**ignoreKey**
+
```

Comment:
I do not see that. Can you explain? You can start by explaining the meaning of this regular expression. Then you can tell which parts of the read value and what part of the expected value are replaced. From this your statement should follow.

Reply:
Very good. I close this point.